plotting_themes.json

```diff
@@ -0,0 +1,34 @@
+{
+    "dark": {
+        "axes.edgecolor": "white",
+        "axes.labelcolor": "white",
+        "xtick.color": "white",
+        "ytick.color": "white",
+        "legend.framealpha": 0.1,
```

Comment:
just my taste ignore if disagree
```suggestion
        "legend.framealpha": 0.3,
```

Reply:
I have in general no strong opinions here, I'll go with whatever @AdrianSosic prefers.

Reply:
same, no opinion

Reply:
I'll double check and choose whichever version I prefer ;)